### PR TITLE
Er 52 google tag manager code.

### PR DESCRIPTION
**Pull request by jdiazarg**
-added fixes to include google tag manager feature.
- to use GTM is needed signup with a gmail account, I got one for testing purposes:
 user: testersonUS@gmail.com  (nothing special with the name, just for test)
pass: Password_
https://tagmanager.google.com

tags used in test:
- container_id:  GTM-PN2WFV and GTM-MHH49G
- analytics: UA-68779712-1

### CODE REVIEW

File: app/config/application/default.json [hunk 1]
```diff
@@ -87,11 +87,13 @@
     "memcached":{
         "type":"Memcached",
         "class":"\erdiko\core\cache\Memcached"
-    }
     },
-    "analytics":{
-        "google":{
-            "id":""
```
Comment (jdiazarg) [inline app/config/application/default.json]: @arroyo @saarmstrong Guys the conficts are part of the changes, here the analytics key is moved to 'google' key.

File: app/themes/bootstrap/theme.json [hunk 2]
```diff
@@ -47,9 +47,12 @@
     "footer": {
         "file": "page/footer"
     },
-    "analytics": {
```
Comment (jdiazarg) [inline app/themes/bootstrap/theme.json]: @arroyo @saarmstrong the same here, we do not need this reference any more.

File: app/themes/bootstrap/theme.json [hunk 3]
```diff
@@ -19,24 +19,24 @@
...
+    },
+    "footer": {
+        "file": "page/footer"
+    },
+    "google_body": {
+        "file": "page/google_body"
+    },
+    "google_header": {
+        "file": "page/google_header"
+    },
```
Comment (saarmstrong) [inline app/themes/bootstrap/theme.json]: this is not valid json.

Comment (jdiazarg) [inline app/themes/bootstrap/theme.json]: Done.

**Pull request merged**